docs/how_to/pytorch_install/pytorch_install.md

```diff
@@ -48,107 +37,89 @@ Follow these steps:
-:name: Installation-Matrix-from-Pytorch
----
-align: center
----
-Installation Matrix from Pytorch
-```
+1. Choose one of the following three options:
 
-To install PyTorch using the wheels package, follow these installation steps:
+   **Option 1:**
 
-1. Choose one of the following options:
-   a. Obtain a base Docker image with the correct user-space ROCm version
-      installed from
-      [https://hub.docker.com/repository/docker/rocm/dev-ubuntu-20.04](https://hub.docker.com/repository/docker/rocm/dev-ubuntu-20.04).
+   Download a base Docker image with the correct user-space ROCm version. See:
+   [https://hub.docker.com/repository/docker/rocm/dev-ubuntu-20.04](https://hub.docker.com/repository/docker/rocm/dev-ubuntu-20.04).
 
-   or
+   **Option 2:**
 
-   b. Download a base OS Docker image and install ROCm following the
-      installation directions in the section
-      [Installation](../../deploy/linux/install.md). ROCm 5.2 is installed in
-      this example, as supported by the installation matrix from
-      <https://pytorch.org/>.
+   Download a base OS Docker image and install ROCm using the directions in the
+   [Installation](../../deploy/linux/install.md) section.
 
-   or
+   **Option 3:**
 
-   c. Install on bare metal. Skip to Step 3.
+   Install on bare metal. If using this method, skip step 2 (starting a Docker container).
 
-      ```bash
-      docker run -it --device=/dev/kfd --device=/dev/dri --group-add video rocm/dev-ubuntu-20.04:latest
-      ```
+   ```bash
+   docker run -it --device=/dev/kfd --device=/dev/dri --group-add video rocm/dev-ubuntu-20.04:latest
+   ```
 
-2. Start the Docker container, if not installing on bare metal.
+2. Start the Docker container (skip this step for bare metal installations).
 
    ```dockerfile
    docker run -it --device=/dev/kfd --device=/dev/dri --group-add video rocm/dev-ubuntu-20.04:latest
    ```
 
-3. Install any dependencies needed for installing the wheels package.
+3. Install the required dependencies for the wheels package.
 
    ```bash
    sudo apt update
    sudo apt install libjpeg-dev python3-dev
    pip3 install wheel setuptools
    ```
 
-4. Install torch, `torchvision`, and `torchaudio` as specified by the installation
-   matrix.
+4. Install torch, `torchvision`, and `torchaudio`, as specified in the _Run this Command_ column of the
+   interactive table on the [PyTorch Start Locally](https://pytorch.org/get-started/locally/) website.
 
    :::{note}
-   ROCm 5.2 PyTorch wheel in the command below is shown for reference.
+   The following command uses the ROCm 5.4.2 PyTorch wheel. If you want a different version of
+   ROCm, modify the command accordingly.
    :::
 
    ```bash
-   pip3 install --pre torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/nightly/rocm5.2/
+   pip3 install --pre torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/nightly/rocm5.4.2/
```

Comment:
`--index-url` option, not `--extra-index-url`
@jithunnair-amd can we remove `torchvision` and `torchaudio`? There are no tests depending on them

Reply:
No, let's keep them to align with the command on pytorch.org, if they don't cause any issues.